tox.ini

```diff
@@ -4,9 +4,19 @@ envlist = py37,flake8
 [testenv]
 deps =
     -rrequirements.txt
+    pytest-cov
 setenv =
  PYTHONPATH=.
-commands = {envpython} {envbindir}/run-ci --ignore=tests -c unittests/pytest.ini {posargs}
+commands = py.test \
```

Comment:
can we add some unit test to check run-ci invocation is not broken?

Reply:
No. If the unittests are working without that, there is no reason to run unittests via `run-ci`.

Reply:
@mbukatov my point is also to test run-ci breakage in CI tests or else we will not catch that. I am fine running unit tests via py.test but we also want to test run-ci is not broken and that was the main reason we added it.

Reply:
@vasukulkarni But that is not a unit test. We added the run-ci there as a hack to make the unit tests work with the new config code. If that is no longer necessary (I haven't rechecked myself), let's remove the hack.

Reply:
@vasukulkarni can you help me understand what specifically you want covered by unit tests here?

Reply:
@zmc the change here remove run-ci to run unittests and uses py.test, I am saying if you can add a unit test that can just invoke run-ci and even check usage that should be fine.

Reply:
@vasukulkarni like this? https://github.com/red-hat-storage/ocs-ci/pull/210/commits/afdae23cc00434c8d69e749eafab894baa3cb489

Reply:
@zmc yep thanks!